src/pages/Clients/index.js

```diff
@@ -0,0 +1,242 @@
+import React, { useState, useEffect } from "react";
+import axios from "../../config/API";
+import { animateScroll as scroll } from "react-scroll";
+import { getAuthorizationHeader } from "../../services/Authentication";
+import { Formik } from "formik";
+import FormMultipleSelect from "../../components/FormComponents/FormMultipleSelect";
+import {
+  realEstateTypes,
+  neighborhoods,
+} from "../../utils/FormHelpers/form-data";
+import { formatCollectionForReactSelect } from "../../utils/FormHelpers/FormatCollectionForReactSelect";
+import { FormTextField } from "../../components/FormComponents/FormTextField";
+import parseJwt from "../../utils/parseJwt";
+
+function AddClient({
+  setNewClientData,
+  newClientData,
+  setShowForm,
+  clientsData,
+  setClientsData,
+}) {
+  const handlePost = async (values) => {
+    const token = window.localStorage.getItem("userToken");
+    const { userId } = parseJwt(token);
+
+    const realEstatesConstructionType = values.realEstatesConstructionType
+      .map((el) => {
+        return el.value;
+      })
+      .join(", ");
+
+    const realEstatesNeighborhood = values.realEstatesNeighborhood
+      .map((el) => {
+        return el.value;
+      })
+      .join(", ");
+
+    setNewClientData({
```

Comment:
Then this call here becomes like this:
```suggestion
    setNewClientData(mappedValues)
```

Reply:
Done.